charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml

```diff
@@ -26,6 +26,21 @@ spec:
     kind: Issuer
   secretName: helm-repository-tls
 ---
+apiVersion: v1
```

Comment:
sounds reasonable

Reply:
what cases are you think where we need to disable it dynamically?
Disabling it would need to change the deployment too. it might be good to remove it when it time to disable it?

Reply:
all of this is already guarded behind 

{{ if .Values.selfHostedRegistry }}